nodejs/eks/package.json

```diff
@@ -44,6 +44,11 @@
         "tslint-config-prettier": "^1.18.0",
         "typescript": "^4.6.2"
     },
+    "overrides": {
+      "@pulumi/pulumi": {
+        "google-protobuf": "3.21.4"
```

Comment:
I only see google-protobuf@^3.5.0 in the yarn lock. Why don't we just initiate an update of that transitive dependency that should resolve it without requiring to hardcode the exact version?

Reply:
Let me try that, I think you're saying I should not be editing package.json constraints and simply update the resolution of google-protobuf to 3.21.4 in the yarn.lock.

Reply:
Yeah, if you remove the google-protobuf section there and run `yarn` again it should install the latest version.